isis/src/tgo/tsts/stitchUnstitch/Makefile

```diff
@@ -0,0 +1,70 @@
+APPNAME =
+
+include $(ISISROOT)/make/isismake.tsts
+
+commands:
+	tgocassis2isis $(TSTARGS)                                                          \
+            from=$(INPUT)/CAS-MCO-2016-11-26T22.50.27.381-PAN-00005-B1.xml           \
+            to=$(OUTPUT)/panframelet.cub > /dev/null;
+
+	spiceinit $(TSTARGS)                                                               \
+            from=$(OUTPUT)/panframelet.cub                                           \
+            ckp=t spkp=t > /dev/null;
+
+	tgocassis2isis $(TSTARGS)                                                          \
+            from=$(INPUT)/CAS-MCO-2016-11-26T22.50.27.381-RED-01005-B1.xml           \
+            to=$(OUTPUT)/redframelet.cub > /dev/null;
+
+	spiceinit $(TSTARGS)                                                               \
+            from=$(OUTPUT)/redframelet.cub                                           \
+            ckp=t spkp=t > /dev/null;
+
+	tgocassis2isis $(TSTARGS)                                                          \
+            from=$(INPUT)/CAS-MCO-2016-11-26T22.50.27.381-BLU-03005-B1.xml           \
+            to=$(OUTPUT)/bluframelet.cub > /dev/null;
+
+	spiceinit $(TSTARGS)                                                               \
+            from=$(OUTPUT)/bluframelet.cub                                           \
+            ckp=t spkp=t > /dev/null;
+
+	tgocassis2isis $(TSTARGS)                                                          \
+            from=$(INPUT)/CAS-MCO-2016-11-26T22.50.27.381-NIR-02005-B1.xml           \
+            to=$(OUTPUT)/nirframelet.cub > /dev/null;
+
+	spiceinit $(TSTARGS)                                                               \
+            from=$(OUTPUT)/nirframelet.cub                                           \
+            ckp=t spkp=t > /dev/null;
+
+	tgocassisstitch $(TSTARGS)                                                         \
+            fromlist=$(INPUT)/cubelist.lis                                           \
```

Comment:
You need to dynamically generate the cube lists because they will contain relative paths and if you run the tests outside of the test directory, they will fail.

Reply:
For the first list, you can just do `ls $(OUTPUT)/*cub` before this call.